### PR TITLE
sending cpu load latency performance test results to big query

### PR DESCRIPTION
**Pull request by lyuxuan**
modify scenario_result_schema.json to accommodate cpu load related statistics.
modify run_performance_tests.py to integrate cpu load latency tests with other performance tests.

### CODE REVIEW

**Comment by ctiller** [comment]
Jenkins: test this please

File: tools/run_tests/run_performance_tests.py [hunk 1]
```diff
@@ -403,6 +406,10 @@ def finish_qps_workers(jobs):
                   action='store_const',
                   const=True,
                   help='Run netperf benchmark as one of the scenarios.')
+argp.add_argument('--server_cpu_load',
```
Comment (ctiller) [inline tools/run_tests/run_performance_tests.py]: Could we just make this an integer argument? Maybe we get interested in 17% CPU load

**Comment by ctiller** [comment]
Can this be merged?

**Comment by lyuxuan** [comment]
#8545
#9124

**Comment by lyuxuan** [comment]
In run_performance_tests.py I changed the JobSepc timeout_seconds from 3*60 to timeout_seconds=12*60

**Pull request merged**